ush/analyses/plot_util.py

```diff
@@ -1624,7 +1624,7 @@ def get_ci_file(stat, input_filename, fcst_lead, output_base_dir, ci_method):
    return CI_file          
 
 def equalize_samples(logger, df, group_by):
-    with open('/u/perry.shafran/test.pkl','wb') as f:
+    with open('/u/emc.vpppg/test.pkl','wb') as f:
```

Comment:
@PerryShafran-NOAA -- one question here. What happens in this script when the job is run by NCO? How can we make this more flexible?

Reply:
I think that Marcel had changed that somewhere.  We'll need to corroborate fixes with him at some point, so this won't be submitted to NCO in this manner.

Reply:
@MarcelCaron-NOAA  -- can you suggest the work-around for using the explicit user name in the `plot_util.py` ush script?